runtime/common/src/weights/rocksdb_weights.rs

```diff
@@ -0,0 +1,95 @@
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2024-08-22 (Y/M/D)
+//! HOSTNAME: `ip-10-173-4-131`, CPU: `Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz`
+//!
+//! DATABASE: `RocksDb`, RUNTIME: `Frequency`
+//! BLOCK-NUM: `BlockId::Number(4413540)`
+//! SKIP-WRITE: `false`, SKIP-READ: `false`, WARMUPS: `2`
+//! STATE-VERSION: `V1`, STATE-CACHE-SIZE: ``
+//! WEIGHT-PATH: ``
+//! METRIC: `Average`, WEIGHT-MUL: `1.3`, WEIGHT-ADD: `0`
+
+// Executed Command:
+//   ./target/release/frequency
+//   benchmark
+//   storage
+//   --state-version=1
+//   --chain=frequency
+//   --base-path=/data
+//   --include-child-trees
+//   --warmups=2
+//   --mul=1.3
```

Comment:
We could use a bigger multiplier if we expect to add more keys until the next update of db weights.